control/statesp.py

```diff
@@ -69,6 +69,19 @@
 
 __all__ = ['StateSpace', 'ss', 'rss', 'drss', 'tf2ss', 'ssdata']
 
+
+def _matrix(a):
+    """_matrix(a) -> numpy.matrix
+    a - passed to numpy.matrix
+    Wrapper around numpy.matrix; unlike that function,  _matrix([]) will be 0x0
```

Comment:
Not clear to my why this is useful?   Can you say a bit here (or in a comment in the code)?

Reply:
I'll expand the commentary; it's just a helper to make it easier to get 0x0 matrices, which are needed for empty state space systems.